### PR TITLE
initial layout of the item screen

### PR DESCRIPTION
**Pull request by KejPat**
Initial layout to item screen and added some functionality to the purchase button

### CODE REVIEW

**Comment by sazzamac** [comment]
Merge conflicts

**Comment by sazzamac** [comment]
This looks so good!! Just a couple of questions:

- What information do you expect to see in the main box? This will probably have it's own dedicated issue, but what would you like to see there? This is the API request for nutella for example: https://world.openfoodfacts.org/api/v0/product/3017620422003.json 
- What is the star icon for?

File: src/screens/ItemScreen.js [hunk 1]
```diff
@@ -0,0 +1,236 @@
...
+            transparent={true}
+            visible={modalVisible}
+            onRequestClose={() => {
+                setModalVisible(false);
+        }} >
+            <View style={styles.modalContainer}>
+                <View style={styles.modalView}>
+                    <Text style={styles.modalText}>Enter number of purchased items: </Text>
+                    {/* input how many items bought */}
+                    <TextInput
```
Comment (sazzamac) [inline src/screens/ItemScreen.js:87]: Is there a numeric counter component with arrows we could use? Not essential but would be nicer from a UIUX perspective

File: src/screens/ItemScreen.js [hunk 2]
```diff
@@ -0,0 +1,236 @@
...
+ },
+
+ modalView: {
+    margin: 20,
+    backgroundColor: "#FAFAFA",
+    borderRadius: 20,
+    padding: 35,
+    alignItems: "center",
+    shadowColor: "#000",
+    shadowOffset: {
```
Comment (sazzamac) [inline src/screens/ItemScreen.js:201]: Love this effect

File: src/screens/ItemScreen.js [hunk 3]
```diff
@@ -0,0 +1,236 @@
...
+    const [modalVisible, setModalVisible] = useState(false);
+    const [itemCounter, setItemCounter] = useState(1);
+
+    // handles if item is favourited or not
+    const favouriteHandler = () => {
+        // TODO: code to add item to favorite list
+        setFavourite(!favourite);
+    }
+
+    // submit the number of purchased items
```
Comment (sazzamac) [inline src/screens/ItemScreen.js:17]: Nice that you added comments, I forgot to

**Review by sazzamac — APPROVED** [review]

File: src/screens/ItemScreen.js (hunk 1 above)
Comment (KejPat) [reply src/screens/ItemScreen.js:87]: Yeah, I was looking into it but thought we could change it later, I just wanted to put something quick in

**Comment by KejPat** [comment]
I'll look into the info needed on a separate issue and look into the json and the star icon is to add the item into the saved section.

**Review by Sajni94 — APPROVED** [review]

**Pull request merged**